blogapi/views.py

```diff
@@ -1,9 +1,12 @@
 from rest_framework import viewsets
 from .models import BlogpotslList
 from .serializers import Blogserilizers
+from rest_framework.pagination import LimitOffsetPagination
 
 # Create your views here.
 
 class BlogApi(viewsets.ModelViewSet):
     queryset = BlogpotslList.objects.all()
     serializer_class = Blogserilizers
+    pagination_class = LimitOffsetPagination
```

Comment:
you don't need this. the default should work without this line

Reply:
fixed